packer/module_capabilities.py

```diff
@@ -0,0 +1,17 @@
+MODULE_CAPABILITIES = [
+    {'name': 'cap_types', 'desc': 'module has its own types and not only operate on existing redis types'},
+    {'name': 'cap_types_limits', 'desc': 'module defines the max value size restrictions for all its types'},
+    {'name': 'cap_multi_key', 'desc': 'module has no methods that operate on multiple keys'},
+    {'name': 'cap_storage', 'desc': 'module has no outside storage in RAM beyond Redis keyspace storage'},
+    {'name': 'cap_replica_of', 'desc': 'module can work with replicaof capability when it is loaded into a source or a destination database'},
+    {'name': 'cap_backup_restore', 'desc': 'module can work with import/export capability'},
+    {'name': 'cap_eviction_expiry', 'desc': 'module is able to handle eviction and expiry without an issue'},
+    {'name': 'cap_reshard_rebalance', 'desc': 'module is able to operate in a database that is resharded and rebalanced'},
+    {'name': 'cap_failover_migrate', 'desc': 'module is able to operate in a database that is failing over and migrating'},
+    {'name': 'cap_wait', 'desc': 'module is able to operate in a database when WAIT is specified with the execution of commands that impact the module data'},
+    {'name': 'cap_multi_exec', 'desc': 'module is able to operate in a database when MULTI/EXEC block is specified with the execution of commands that impact the module data'},
+    {'name': 'cap_persistence', 'desc': 'module is able to operate in a database when database chooses any of the AOF or SNAPSHOT persistence options'},
+    {'name': 'cap_replication', 'desc': 'module is able to operate in a database with Replication enabled or disabled'},
```

Comment:
all modules work with replication, it's not a special capability

Reply:
Doesn't module have to explicitly call RedisModule_Replicate to replicate its commands?
Suppose a module doesn't do that, do we still want to enable replication on its host db?

Reply:
It's a defect if it doesn't and it needs to be fixed. It's like not binding a command. It's not something we will allow, and should discover in QA